### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Oifox** [BryanGIG/LibDumper#3]
This Dumper dump standoff 2 game? version game 0.15.11

**Comment by BryanGIG** [comment]
no, not yet but i already figure out how to get unencrypted libil2cpp.so and global-metadata from this game.
maybe i will added it into my LibDumper in the future
